documentation/public/user-guide/npm-package/using-queries.md

```diff
@@ -0,0 +1 @@
+# Using Queries
```

Comment:
Could we drop the queries from this PR? This is already merged upstream and the changes here are conflicting

Reply:
Opened https://github.com/AntonyBlakey/slang/pull/5 to facilitate that

Reply:
done